lib/app/components/pages/search_filter/views/search_filter_view.dart

```diff
@@ -63,44 +78,57 @@ class SearchFilterView extends GetView<SearchFilterController> {
-                  selectedItems: [
-                    Text(controller.belongCommunity),
-                  ],
+                  selectedItems: controller.belongCommunities.map(
+                    (community) => Text(community)
+                  ).toList(),
                   label: 'コミュニティ',
+                  emptyChild: const Text(
+                    'こだわらない',
+                    textAlign: TextAlign.left,
+                    style: TextStyle(
+                      color: Colors.grey,
+                      fontWeight: FontWeight.bold,
+                    ),
+                  ),
                   onTap: () => Get.toNamed(Routes.searchSelectCommunity,),
                 ),
-
-                Row(
+                Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
                   children: [
-                    SizedBox(
-                      width: context.width/2,
-                      child: Padding(
-                        padding: fieldPadding,
-                        child: Obx(()=>GenderSelectButton(
-                          label: "性別",
-                          menuMaxHeight: MediaQuery.of(context).size.height / 3,  // ドロップダウン表示は画面高さの1/3まで
-                          onChanged: controller.changeGender,
-                          value: controller.selectedGender,
-                        ),),
+                    const Padding(
+                      padding: EdgeInsets.only(top: 8, left: 8, right: 8),
+                      child: Text(
+                        '性別',
+                        textAlign: TextAlign.left,
+                        style: TextStyle(
+                          fontWeight: FontWeight.bold,
+                        ),
                       ),
                     ),
-                    SizedBox(
-                      width: context.width/2,
-                      child: Padding(
-                        padding: fieldPadding,
-                        child: Obx(()=> AgeSelectButton(
-                          ageLimMax: targetAgeLimitMax,
-                          ageLimMin: targetAgeLimitMin,
-                          label: "年代",
-                          menuMaxHeight: MediaQuery.of(context).size.height / 3,  // ドロップダウン表示は画面高さの1/3まで
-                          onChanged: controller.changeAge,
-                          value: controller.selectedAge,
-                        ),),
-                      ),
+                    Row(
+                        children: [
+                          Obx(() => _buildGenderCheckBox(
+                            context: context,
+                            values: controller.selectedGender,
+                            sidePadding: 8,
+                            onChanged: (gender, value){
+                              controller.changeGender(gender, value);
+                            },
+                          ),
+                          ),
+                        ]
                     ),
```

Comment:
```suggestion
      Obx(() => _buildGenderCheckBox(
                   context: context,
                   values: controller.selectedGender,
                   sidePadding: 8,
                   onChanged: (gender, value){
                   controller.changeGender(gender, value);
                     },
                   ),
                 ),
```
Rowはいらない気がするので削除しましょう